packages/frontpage/lib/schema.ts

```diff
@@ -124,6 +124,12 @@ export const BetaUser = sqliteTable("beta_users", {
   did: did("did").notNull().unique(),
 });
 
+export const AdminUser = sqliteTable("admin_users", {
+  id: integer("id").primaryKey(),
+  createdAt: dateIsoText("created_at").notNull(),
```

Comment:
`updated_at` not needed? I feel like it should be here if only for consistency.

Reply:
we are manually adding these to the db and the beta user table doesn't have them, I don't think we really care abut updated at in this instance tbh

we will when we have an actual users table someday